plugins/woocommerce/uninstall.php

```diff
@@ -65,15 +66,17 @@
 	// Delete usermeta.
 	$wpdb->query( "DELETE FROM $wpdb->usermeta WHERE meta_key LIKE 'woocommerce\_%';" );
 
-	// Delete posts + data.
+	// Delete our data from the post and post meta tables, and remove any additional tables we created.
 	$wpdb->query( "DELETE FROM {$wpdb->posts} WHERE post_type IN ( 'product', 'product_variation', 'shop_coupon', 'shop_order', 'shop_order_refund' );" );
```

Comment:
offtopic- should we even do this during uninstall? i don't think that we give any warning that your data will be deleted

Reply:
By default, we don't do this: for these lines to be executed, the site owner has to modify `wp-config.php` (or other suitable location) and define constant `WC_REMOVE_ALL_DATA` as `true` (see [line 36](https://github.com/woocommerce/woocommerce/blob/ce3ce7cb869e9a50419b32bf60f6dd6d53195210/plugins/woocommerce/uninstall.php#L31-L36)). So, it's a very deliberate choice and it would be hard for someone to do unintentionally.